src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/volume/sdk/AzureVolumeOperationSDKTest.java

```diff
@@ -0,0 +1,91 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.volume.sdk;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.microsoft.azure.management.resources.fluentcore.model.Indexable;
+
+import ch.qos.logback.classic.Level;
+import cloud.fogbow.ras.constants.Messages;
+import cloud.fogbow.ras.core.LoggerAssert;
+import cloud.fogbow.ras.core.plugins.interoperability.azure.AzureTestUtils;
+import rx.Completable;
+import rx.Observable;
+import rx.Scheduler;
+import rx.schedulers.Schedulers;
+
+public class AzureVolumeOperationSDKTest {
+
+    private AzureVolumeOperationSDK operation;
+    private LoggerAssert loggerAssert;
+
+    @Before
+    public void setup() {
+        // The scheduler trampoline makes the subscriptions execute 
+        // in the current thread
+        Scheduler scheduler = Schedulers.trampoline();
+        this.operation = Mockito.spy(new AzureVolumeOperationSDK());
+        this.operation.setScheduler(scheduler);
+        this.loggerAssert = new LoggerAssert(AzureVolumeOperationSDK.class);
+    }
+
+    // test case: When calling the subscribeCreateDisk method and the
+    // observable executes without any error, it must verify than returns the right
+    // logs.
+    @Test
+    public void testSubscribeCreateDiskSucessfully() {
```

Comment:
Successfully misspelled in the name of the method.

Reply:
Fixed.